app/helpers/spree/base_helper_decorator.rb

```diff
@@ -1,17 +1,19 @@
 Spree::BaseHelper.class_eval do
   def display_volume_price(variant, quantity = 1, user = nil)
-    options = SolidusVolumePricing::PricingOptions.new(quantity: quantity, user: user)
-    SolidusVolumePricing::Pricer.new(variant).price_for(options).to_html
+    price_display(variant, quantity: quantity, user: user).price_string
   end
 
   def display_volume_price_earning_percent(variant, quantity = 1, user = nil)
     variant.volume_price_earning_percent(quantity, user).round.to_s
   end
 
   def display_volume_price_earning_amount(variant, quantity = 1, user = nil)
-    Spree::Money.new(
-      variant.volume_price_earning_amount(quantity, user),
-      currency: Spree::Config[:currency]
-    ).to_html
+    price_display(variant, quantity: quantity, user: user).earning_string
```

Comment:
+1 for creating an object to handle this.

However replacing `price_display` with `SolidusVolumePricing::PriceDisplay` actually will do the same thing. They take the same arguments and the method has no logic, so we win on clarity by just explicitly referencing the class.